src/features/profile/HouseKeepingCard.js

```diff
@@ -27,13 +30,23 @@ const HouseKeepingCard = () => {
     exportAllEntriesText = `No entries to export`
   }
 
+  // let FATHOM_EXPORT = "Pieces_Of_Eight"
+
+  // const trackGoal = () => {};
+
   const exportAllEntries = async (event) => {
     setIsPending(true)
     event.preventDefault()
 
+    // The intent of the user is to export her fistful_of_Pieces_Of_Eight
+    trackGoal(FATHOM_EXPORT)
+    console.log("Track fathom goal", {FATHOM_EXPORT})
```

Comment:
**suggestion:** Remove as this is logged by the trackGoal function itself

Reply:
A. remove line 41 + 42 + 43 ? or
B. remove line 42  ? 

I go with B.
😸

Reply:
suggestion: Remove console.log #302
3743d24